src/test/resources/cucumber/features/integrationTests/jira/publish-processing.feature

```diff
@@ -47,57 +49,67 @@ Feature: Check integration tests of publish processing
-      | [High,Medium]   | 10                           | 2                            | 9                              | 3                              | 4                           | 2                           | 20                           | 1                            | 10                            | 9                               | 0                            | 0                             |
-      | [High,Low]      | 10                           | 2                            | 9                              | 3                              | 4                           | 2                           | 20                           | 1                            | 10                            | 0                               | 4                            | 0                             |
-
-
-  @Target_update_Issue
-  Scenario: publish updated issue to JIRA
+      | Filter_Severity | High_Vulnerabilities_Expected | Medium_Vulnerabilities_Expected | Low_Vulnerabilities_Expected  | Info_Vulnerabilities_Expected  |
+      | High            | 2                             | 0                                 | 0                           | 0                              |
+      | High,Medium    | 2                             | 10                                | 0                           | 0                              |
+      | High,Low       | 2                             | 0                                 | 19                          | 0                              |
+
+  @Integration
+  @Update_issue
+  Scenario: updating an existing JIRA issue during publish
+    Given target is JIRA
+    And there is an existing issue
+    And SAST results contain 1 finding
+    When publishing results to JIRA
+    Then JIRA still contains 1 issue
+    And issue ID hasn't changed
+    And issue's updated field is set to a more recent timestamp
+    And issue has the same vulnerability type and filename
+
+  @Integration
+  @Update_issue @Negative_test
+    # Change scenario name
+  Scenario: publish updated issue to JIRA with different parameters
     # Note - to update an issue, the vulnerability & filename fields must match
     Given target is JIRA
     And   there is an existing issue
     When  publishing same issue with different parameters
     Then  original issues is updated both with 'last updated' value and with new body content
 
-
-  @Target_update_Issue @Negative_test
-  Scenario: publish updated issue to JIRA with missing parameters
-    # Note - to update an issue, the vulnerability & filename fields must match
+  @Integration
+  @Close_issue
+  Scenario: publish closed issue to JIRA
     Given target is JIRA
     And   there is an existing issue
-    When  publishing same issue with missing parameters
-    Then  original issues is updated both with 'last updated' value and with new body content
+    When  all issue's findings are false-positive
```

Comment:
Alexey, this scenario was made by Nimrod.
It is the same as in develop.
We see this scenario here because of the rebase I made to my branch over develop.
That is why  I don't want (or need)  to touch it.
Thanks